go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/fission/fission
 
+go 1.15
```

Comment:
Can you please remove this? I think we have some dependency on earlier versions

Reply:
Done!